server/database.py

```diff
@@ -62,3 +66,25 @@ def delete_tag_by_id(db, id):
     if mongo_string == "null":
         return ({"Error": "Resource Does Not Exist"}, 404)
     return ("Delete Successful", 200)
+
+# create_subscriber inserts a subscriber id into the database for the tag with the given tag id 
+def create_subscriber(db, tag_id, subscriber_id):
+    query_results = db.tags.update_one(
+        {'snowflake_id': tag_id},
+        {'$addToSet': {
+            'subscribers': subscriber_id
+            }
+        }
+    )
+    #the query results when the id already exists within the array 
+    if query_results.modified_count == 0:
+        return({'Success': 'Resource Already Exists'}, 200)
```

Comment:
Should this be a 200 success?

Reply:
I think so, a 201 would indicate a new resource is being created but this is just confirming that it already exists. I think a 4xx code is incorrect since that would imply there was an error

Reply:
I guess its kinda up to us huh, i was reading about it and seems theres some debate on the "proper" usage, but really its just up to the devs i guess to try to provide the best context? 

as long as we understand that i guess the request _was_ successful, its just nothing was changed? lol idk, not gonna block it for this